### ISSUE TITLE
Publish to Maven central. 

### ISSUE THREAD

**Issue by hohonuuli** [mbari-org/commons#1]
For reference, here's the vcr4j [issue for publishing on sonatype](https://issues.sonatype.org/browse/OSSRH-87424)

**Comment by hohonuuli** [comment]
Submitted request to sonatype: https://issues.sonatype.org/browse/OSSRH-88432

**Comment by hohonuuli** [comment]
I've done a release to Sonatype, I *think* it worked. I'm using the nexus plugin to automatically release if all goes well. I tried with a few version but got an error for scommons (no javadoc attached). So I added a dummy Java class and reran it. Nothing is appearing in the staging Or at least nothing has appeared in the release "Staging Repositories" section at <https://s01.oss.sonatype.org>, which happens if something goes run. So I'm guessing it worked. I'll check again later to see if the artifacts appear in maven central.
